modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm

```diff
@@ -33,14 +33,14 @@
 	belt = /obj/item/pda/security
 	ears = /obj/item/radio/headset/headset_medsec
 	uniform = /obj/item/clothing/under/rank/security/peacekeeper/security_medic
-	gloves = /obj/item/clothing/gloves/color/latex
+	gloves = /obj/item/clothing/gloves/color/latex/nitrile
 	shoes = /obj/item/clothing/shoes/combat/peacekeeper
 	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
 	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/security_medic
-	l_hand = /obj/item/storage/firstaid/advanced
+	l_hand = /obj/item/storage/firstaid/brute
 	head = /obj/item/clothing/head/beret/sec/peacekeeper/security_medic
 
-	backpack_contents = list(/obj/item/melee/classic_baton/telescopic, /obj/item/storage/box/gunset/security_medic, /obj/item/defibrillator/compact/loaded)
```

Comment:
Leave the defib in as it's part of their job to revive security in the field. Imagine they're on an away mission down under icebox, this will save them.

Reply:
No. That's the whole idea. 

They have to care enough about their job to print one and load a cell like everyone else if they want to bypass the role that is intended to have access to defibs. Do you know which medical roles walk off the shuttle with a defibrillator? None of them. None at all. The only compact defib at round start is in the CMO's locker, and he has to at least go into his office and retrieve it. Medical doctors have access to a limited number of full size defibs in their equipment lockers, and paramedics do not have access to those.

Why even have a research tree if we are going to spawn in the items it locks anyway? Why have any sense of round progression at all? Why not just start every round with all tech unlocked, and spawn every job with it's advanced tools? The station would only need one or two medical staff total if we just gave them every end game tool and access to all equipment.

Going further, why does the brig medic need to do revivals? Can they not cart people back to medbay? Medics should not be doing that job at all, but this PR leaves the door open for them to obtain a compact defib the same way anyone else who wants one can get one, by printing it from the medical department tech fab, after it has been researched.

Without a meaningful skills system, the best way to make players at least pretend to stick within the lanes of their job is to limit their access to other job's equipment, especially access to late game items at round start.

Reply:
they could also ask cargo nicely for the 5000 credit combat defib they can buy, if they really insist.

Reply:
As an aside, I think that combat defib should not be for sale in cargo, and neither should the beam. Those are supposed to be an ERT and antag item. That combat defib should also get buffed so that it actually stunlocks people like it did on the old base.

Reply:
Ehhh, the beam is plenty expensive and I think the crates are CMO only unless you personal buy them, something doctors like to do with bounties. It keeps medbay productive (cargo mains LOVE this) on very slow shifts and gives them a goal to work towards.